examples/resources/rafay_blueprint/resource.tf

```diff
@@ -135,10 +135,9 @@ resource "rafay_blueprint" "blueprint" {
       name = "addon1"
       version = "v0"
     }
-    custom_addons {
-      depends_on = ["addon1"]
-      name = "addon2"
-      version = "v0"
+    components_criticality {
```

Comment:
Don't delete `custom_addons` ( that is needed as it contains how to add depends_on)..

Reply:
Addressed your comment @krishnarafay